.github/workflows/bootstrap_region.yml

```diff
@@ -0,0 +1,94 @@
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        options:
+          - beta
+          - prod
```

Comment:
We don't need to balance layers in beta. In fact, this can cause problems because the layers are completely unbalanced in beta. Please remove beta.

Reply:
this is mostly for testing in beta, at most it'll add a bunch of junk layer versions in beta that we don't care about.